engine/lite/runtime/src/main/scala/pl/touk/nussknacker/engine/lite/TestRunner.scala

```diff
@@ -48,11 +50,27 @@ class InterpreterTestRunner[F[_] : InterpreterShape : CapabilityTransformer : Ef
 
     //TODO: probably we don't need statics here, we don't serialize stuff like in Flink
     val collectingListener = ResultsCollectingListenerHolder.registerRun(variableEncoder)
-
     //in tests we don't send metrics anywhere
     val testContext = LiteEngineRuntimeContextPreparer.noOp.prepare(testJobData(process))
     val componentUseCase: ComponentUseCase = ComponentUseCase.TestRuntime
 
+    val expressionEvaluator: (Expression, Parameter, Context, NodeId) => Validated[NonEmptyList[PartSubGraphCompilationError], (String, AnyRef)] = {
```

Comment:
1. Let's replace passing around Context with `Context("dumb", Map.empty)`, we don't need "production" context to evaluate this parameters
2. Use `ValidateNel` to avoid nested types

Reply:
Done :+1: